src/MfaVerifyController.js

```diff
@@ -100,6 +115,15 @@ function (Okta, Checkbox, BaseLoginController, TOTPForm, YubikeyForm, SecurityQu
           this.options.appState.get('isMfaRequired')) {
         return true;
       }
+      // update auto push cookie after user accepts Okta Verify MFA
+      if (this.options.factorType == 'push') {
+        if (this.settings.get('features.autoPush') && !!this.model.get('autoPush')) {
+          CookieUtil.setAutoPushCookie(this.options.appState.get("userId"));
```

Comment:
I'm guessing lint will break, but you shouldn't use double quotes here and in line 123